### ISSUE TITLE
Unzip command missing from latest tomcat:9-jdk11

### ISSUE THREAD

**Issue by jpl-jengelke** [docker-library/tomcat#267]
Attempting to run ‘unzip’ in the container results in an error. This impacts deployments that use the container for various operations involving decompressing ZIP archives.

**Comment by tianon** [comment]
This is an unfortunate consequence of #265 :disappointed:

**Comment by jpl-jengelke** [comment]
What makes this a "usability question" per se? I think there should be a fundamental expectation that basic tools are included by default in build images. In fact, the Open Group Base Specification lists a number of required POSIX utilities. Now `unzip` may not be part of that but in usage it's the most common archiving application, I believe. One would reasonably expect basic archiving utilities to be installed in a Java-centered container that deals in JARs/WARs/etc., especially since the Java compilation products are ZIP-based.

**Comment by wglambert** [comment]
https://github.com/docker-library/openjdk/issues/35#issuecomment-102191500
You're right, it was in the image because of its usefulness and being so small at less than 1 mB

**Comment by abesnier** [comment]
Ow! And I was wondering why extracting tar.xz files failed!

**Comment by jpl-jengelke** [comment]
Any traction on this?

**Comment by tianon** [comment]
Sorry for the delay.  I think I've got better words for my position on this now than I did the last time I looked at this. :sweat_smile:

Originally, we did not _intentionally_ include `unzip`, so while it was unintentionally removed as well, I don't believe there's any functionality in Tomcat itself which invokes `unzip` directly (would love a pointer to the relevant functionality if so!), so we unfortunately do not plan to add it back (unless there's some functionality in Tomcat itself which invokes it). :heart:
